internal/services/loadbalancer/loadbalancer_resource.go

```diff
@@ -121,6 +131,16 @@ func loadBalancer() *pluginsdk.Resource {
 							Set: pluginsdk.HashString,
 						},
 
+						"outbound_rules_id": {
```

Comment:
should this be rule ids?
```suggestion
						"outbound_rule_ids": {
```